### PR TITLE
Spark 501675 upgrade node version

### PR DESCRIPTION
**Pull request by Kesari3008**
Updating node version from 14 to 20.13.1.
Along with the node version upgrade, upgraded following packages:

node-sass
rollup-plugin-scss
react-test-renderer

Updated eslint configuration to resolve the path of the module facing issue with import/no-unresolved rule.
Updated rollup config to resolve the path too

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "main": "dist/umd/webex-components.umd.min.js",
   "scripts": {
     "build": "rollup -c",
-    "linter": "eslint src/ --ext .js,.jsx",
+    "linter": "eslint src/",
```
Comment (sreenara) [inline package.json:10]: Do we need this change?

File: package.json [hunk 2]
```diff
@@ -114,7 +117,8 @@
     "verbose": true,
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
-      "\\.(css|less|scss)$": "identity-obj-proxy"
+      "\\.(css|less|scss)$": "identity-obj-proxy",
+      "^@webex/component-adapter-interfaces(.*)$": "<rootDir>/node_modules/@webex/component-adapter-interfaces/dist/webex-component-adapter-interfaces.umd"
```
Comment (sreenara) [inline package.json:121]: Is this still required after the change in rollup config?

File: src/components/WebexInMeeting/WebexInMeeting.stories.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ export default {
   component: WebexInMeeting,
 };
 
-const Template = (args) => {
+const Template = (...args) => {
```
Comment (sreenara) [inline src/components/WebexInMeeting/WebexInMeeting.stories.js:10]: Why are these changes needed in the stories?

File: src/components/WebexSettings/Tabs.jsx [hunk 4]
```diff
@@ -97,6 +97,7 @@ Tabs.propTypes = {
   onSelect: PropTypes.func.isRequired,
   selected: PropTypes.string.isRequired,
   style: PropTypes.shape(),
+  // eslint-disable-next-line react/forbid-prop-types
```
Comment (sreenara) [inline src/components/WebexSettings/Tabs.jsx:100]: Why is this eslint disable needed?

File: src/components/generic/Button/Button.jsx [hunk 5]
```diff
@@ -54,6 +54,7 @@ export default function Button({
         tabIndex={tabIndex}
         ref={buttonRef}
         style={{height: size, ...style}}
+        // eslint-disable-next-line react/no-unknown-property
```
Comment (sreenara) [inline src/components/generic/Button/Button.jsx:57]: Why was this needed?

File: src/components/hooks/useMembers.js [hunk 6]
```diff
@@ -35,7 +35,11 @@ export default function useMembers(destinationID, destinationType) {
       console.error(error.message);
     };
     const onMembers = (memberList) => {
-      setMembers([...memberList]);
+      if (Array.isArray(memberList)) {
```
Comment (sreenara) [inline src/components/hooks/useMembers.js:38]: Why was this change needed?

**Review by sreenara — CHANGES_REQUESTED** [review]
Please see the comments

File: src/components/WebexInMeeting/WebexInMeeting.stories.js (hunk 3 above)
Comment (Kesari3008) [reply src/components/WebexInMeeting/WebexInMeeting.stories.js:10]: These were giving errors

File: src/components/WebexSettings/Tabs.jsx (hunk 4 above)
Comment (Kesari3008) [reply src/components/WebexSettings/Tabs.jsx:100]: While committing was observing error for these lines

File: src/components/generic/Button/Button.jsx (hunk 5 above)
Comment (Kesari3008) [reply src/components/generic/Button/Button.jsx:57]: While committing was observing error for these lines

File: src/components/hooks/useMembers.js (hunk 6 above)
Comment (Kesari3008) [reply src/components/hooks/useMembers.js:38]: While committing was observing error for these lines

File: package.json (hunk 2 above)
Comment (Kesari3008) [reply package.json:121]: This change is the one which fixes the module resolution error

File: src/components/WebexSettings/Tabs.jsx (hunk 4 above)
Comment (sreenara) [reply src/components/WebexSettings/Tabs.jsx:100]: So shouldn't we fix the root cause instead of blocking the eslint error? @Kesari3008

File: package.json (hunk 1 above)
Comment (Kesari3008) [reply package.json:10]: We do not need to specify extension separately

File: src/components/generic/Button/Button.jsx (hunk 5 above)
Comment (Kesari3008) [reply src/components/generic/Button/Button.jsx]: removed it

File: src/components/WebexSettings/Tabs.jsx (hunk 4 above)
Comment (Kesari3008) [reply src/components/WebexSettings/Tabs.jsx]: removed it

File: package.json (hunk 2 above)
Comment (Kesari3008) [reply package.json:121]: While running tests, jest started to fail with module resolution error so added moduleNameMapper to fix that

**Review by sreenara — APPROVED** [review]
